src/api/hook/admin.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from 'react-query';
+import { getCommentList, getUserList } from '../fetcher/admin';
+import { IRequestCommentList, IRequestUserList } from '../types/admin';
+
+export const ADMIN_QUERY_KEY = {
+  USERLIST: 'USERLIST',
+  COMMENTSLIST: 'COMMENTSLIST',
```

Comment:
`LIST` 안붙이고 복수형으로 써도 될거 같아요!
```suggestion
  USERS: 'USERS',
  COMMENTS: 'COMMENTS',
```